ui/StatusQ/src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -418,7 +418,7 @@ Item {
                 icon.name: model.icon
                 icon.color: Theme.palette.userCustomizationColors[root.colorIdForPubkeyGetter(model.pubKey)]
                 status: model.onlineStatus
-                ringSettings.ringSpecModel: root.ringSpecModelGetter(model.pubKey)
```

Comment:
You might now remove the toplevel `property var ringSpecModelGetter` and adjust the usages then